thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -103,6 +103,29 @@ export default Ember.Controller.extend({
   },
 
   /**
+   * Setting default sensitivity if selectedSensitivity is undefined
+   * @returns {String}
+   */
+  sensitivityWithDefault: Ember.computed(
+    'selectedSensitivity', 
+    'selectedGranularity',
+    function() {
+      let {
+       selectedSensitivity, 
+       selectedGranularity
+      } =  this.getProperties('selectedSensitivity', 'selectedGranularity');
+
+      if (!selectedSensitivity) { 
+        selectedSensitivity = (selectedGranularity === '5_MINUTES')
```

Comment:
The selected granularity for minutely metrics can be '5_MINUTES' or 'MINUTES' or '1_MINUTES', is there a better way that we can represent minutely metrics' granularities? Or here we can use "HOURS" or "DAYS" to represent the other two cases and set default value to be "Sensitive"?

Reply:
Good point. will do the reverse per your suggestion. 
Btw, any reason why this is not consistent ?

Reply:
It depends on the data source. For ingraph it can be either 1_MINUTE or 5_MINUTE, for sitespeed metrics I think they from a different data source (harrier) and being imported to pinot. Their granularity are being imported as "MINUTE". Probably needs to unify that.